src/components/ModalBuyMealNew/ModalBuyMealPool.tsx

```diff
@@ -59,35 +88,7 @@ export const Modal = (props: Props) => {
       </div>
       <p className={styles.description}>
         {t('buyMealPool.description.weAre')}
-        <a
-          href="https://www.apexforyouth.org/"
-          target="_blank"
-          rel="noopener noreferrer"
-          className={styles.link}
-        >
-          {' '}
-          APEX for Youth
-        </a>
-        ,
-        <a
-          href="http://www.lajornadany.org/"
-          target="_blank"
-          rel="noopener noreferrer"
-          className={styles.link}
-        >
-          {' '}
-          La Jornada Food Pantry
-        </a>
-        ,
-        <a
-          href="https://www.cpc-nyc.org/"
-          target="_blank"
-          rel="noopener noreferrer"
-          className={styles.link}
-        >
-          {' '}
-          the Chinese-American Planning Council{' '}
-        </a>
+        { Distributor }
```

Comment:
I think if you want, you can change `Distributor` to a lambda that returns that body, and add it as a component here like `<Distributor></Distributor>` -- up to you